src/pybind/mgr/dashboard/controllers/rgw.py

```diff
@@ -330,6 +367,32 @@ def delete(self, bucket, purge_objects='true', daemon_name=None):
             'purge-objects': purge_objects
         }, json_response=False)
 
+    @RESTController.Collection(method='PUT', path='/setEncryptionConfig')
+    @allow_empty_body
+    def setEncryptionConfig(self, encryption_type=None, kms_provider=None, auth_method=None,
```

Comment:
```suggestion
    def set_encryption_config(self, encryption_type=None, kms_provider=None, auth_method=None,
```
we should stick to snake_case in python code (there are more instances of this in these changes)